### PR TITLE
Added new test cases from Demographics Module

### PR DESCRIPTION
**Pull request by dipalith**
1.	 All blanks lines are removed from all spec file.
2.      Corrected indentation for all pages and spec file
3.	Modal text method for popup declared commonly in one page and reuse it in spec as per need
4.     One test script added from manual testing set (KIOS-1521)
5.     Added API post method in command.js for adding insurance card.
Please approve and merge.

### CODE REVIEW

File: cypress/e2e/kiosk/pageObjects/pages/demographic.page.js [hunk 1]
```diff
@@ -4,242 +4,303 @@ import ReviewDemographicsPageData from "../../specs/ui/review.demographicsPage.T
...
+    return this;
+  }
+  static clickOnCancelButton() {
+    const button = cy.get('[data-testid="reset"]');
+    button.click();
+    return this;
+  }
+  static editCommunicationBtn() {
+    const button = cy.get('[data-testid="communication"]');
+    button.click();
```
Comment (RemaKoul) [inline cypress/e2e/kiosk/pageObjects/pages/demographic.page.js:112]: please remove any unnecessary extra lines

**Review by RemaKoul — CHANGES_REQUESTED** [review]
Slight changes suggested, please add those so that we can merge

File: cypress/e2e/kiosk/pageObjects/pages/demographic.page.js [hunk 2]
```diff
@@ -4,242 +4,303 @@ import ReviewDemographicsPageData from "../../specs/ui/review.demographicsPage.T
...
+      { timeout: Cypress.env("elementTimeout") };
+    return this;
+  }
 
+  static fillInvalidFormatDob(value) {
+    cy
+      .get('[data-testid="patientInfo-dateOfBirth"]')
+      .click()
+      .clear()
+      .type("3/01/0"),
```
[the remark this answers is not in the document]
Comment (dipalith) [reply cypress/e2e/kiosk/pageObjects/pages/demographic.page.js:190]: Will update and share.

File: cypress/e2e/kiosk/pageObjects/pages/demographic.page.js (hunk 1 above)
Comment (dipalith) [reply cypress/e2e/kiosk/pageObjects/pages/demographic.page.js:112]: Will update and share corrected files.

File: cypress/e2e/kiosk/pageObjects/pages/demographic.page.js [hunk 3]
```diff
@@ -4,242 +4,303 @@ import ReviewDemographicsPageData from "../../specs/ui/review.demographicsPage.T
...
+      .clear()
+      .type("35/01/00"),
+      { timeout: Cypress.env("elementTimeout") };
+    return this;
+  }
+  static fillDoesNotHaveEnoughDigitDob() {
+    cy
+      .get('[data-testid="patientInfo-dateOfBirth"]')
+      .click()
+      .clear()
```
[the remark this answers is not in the document]
Comment (dipalith) [reply cypress/e2e/kiosk/pageObjects/pages/demographic.page.js:179]: Will pass through data file and share corrected files

File: cypress/e2e/kiosk/pageObjects/pages/demographic.page.js (hunk 1 above)
Comment (dipalith) [reply cypress/e2e/kiosk/pageObjects/pages/demographic.page.js:109]: Updated in new push. please heck

**Review by RemaKoul — APPROVED** [review]
feedback incorporated  . Merging changes . Thanks @dipalith

**Pull request merged**